.gitignore

```diff
@@ -31,6 +31,7 @@ tmux-server-*
 coverage/*
 spec/examples.txt
 wraith/now/*
+wraith/cookies.txt
```

Comment:
What does this do?

Reply:
That's where phantom stores its cookies file. Given that sticks around between tests unless you manually delete it I didn't want git picking it up xD